### ISSUE TITLE
Free API deprecation

### ISSUE THREAD

**Issue by dgw** [sopel-irc/sopel-twitter#40]
In one week, Twitter plans to shut off free API access:

```
01:11:36 <+xrl> Oh no. https://twitter.com/TwitterDev/status/1621026986784337922
01:11:36 <+Sopel> [Twitter] Twitter Dev (@TwitterDev): Starting February 9, we will no longer support free access to the Twitter API, both v2 and v1.1. A paid basic tier will be available instead 🧵 | 7775 RTs | 2880 ♥s | Posted: [05] Thursday, February 02, 07:05:25+0100 (CET)
```

Migrating to v2 no longer matters, I guess. #29 (scraping) all the way?

_(Of course, Elno could change tacks on this ten times in the next week. We'll just have to see.)_

**Comment by dgw** [comment]
@xnaas I swear I mentioned that somewhere, but yesterday was pretty busy IRL so maybe I imagined doing it 😂

Nitter could be The Way™ now. This would also be a perfect excuse to restructure the plugin to entry-point and switch package names (`sopel_modules.twitter` ➡ `sopel-twitter`).

**Comment by dgw** [comment]
Well yes, choosing a Nitter instance would be an issue. Aside from (a contributor to) the project hosting one FBO all Sopel users, you're right. That's even without accounting for all the other instances that users might post links to (even if the URL patterns are fairly predictable…)

I do wish one of the projects I've seen using the bootleg API would include some documentation, but there's a Python library I found to poke at, and it doesn't even look abandoned (yet): https://pypi.org/project/tweety-ns/

**Comment by dgw** [comment]
Started work in a `tweety-migration` branch, but am currently blocked on mahrtayyab/tweety#11 which seems like a major blocker for maintaining the plugin's existing behavior.

**Comment by dgw** [comment]
Resolved by #42
